### ISSUE TITLE
Parameter for clear text password

### ISSUE THREAD

**Issue by lipkau** [philips-software/powershell-cf-api#25]
Passing password as clear text is a bad practice (as the exceptions for PSScriptAnalyzer point out).
I would recommend changing to a `[PSCredential]` object, which is more "powershelly" and you can still fetch the user and password out of the object

**This is a breaking change, as it will change the signature of the function**

**Comment by marklindell** [comment]
I agree. When I wrote it not understand this and I also did not have PSScriptAnalyzer integrated. Thanks for the issue.

**Comment by lipkau** [comment]
will submit the PR once #23 is merged
